tapgarden/caretaker.go

```diff
@@ -785,7 +785,7 @@ func (b *BatchCaretaker) stateStep(currentState BatchState) (BatchState, error)
 		case strings.Contains(err.Error(), "already exists"):
 			break
 
-		case err != nil:
+		default:
```

Comment:
isn't this equiv to the above? A non-nil error ends up here w/ the prior version that had explicit err check. A commit body message could help here re rationale/understanding.